react_demo/index.html

```diff
@@ -7,8 +7,8 @@
   </head>
   <body>
     <div id="c5t-root">Loading...</div>
-    <script src="./vendor/react-0.13.3-build/react.js"></script>
-    <script src="./vendor/react-0.13.3-build/JSXTransformer.js"></script>
+    <script src="./vendor/react/react.js"></script>
```

Comment:
Damn, that's why I don't like copying files around and changing the copies a little.